### PR TITLE
Update steps for running WinAppDriver autonomously

### PR DESCRIPTION
**Pull request by smilinrobin**
The steps in the current documentation leave ambiguity for the reader to figure out setup steps in a CI/CD kind of pipeline.
Adding details around batch files and Task Schedulers to run winappdriver autonomously would help automation/testing teams to setup Winappdriver in a complementary style to the CI/CD flow.  
@timotiusmargo @hassanuz  @licanhua @kat-y  @DHowett - Kindly review and approve the PR as necessary .

### CODE REVIEW

**Comment by vagisha-nidhi195** [comment]
@smilinrobin Does the solution to pass the desktop session to a console session work? I have been able to connect with winappdriver on the console session but wehn clicking any element it results in the following error:
```
{"status":13,"value":{"error":"unknown error","message":"An unknown error occurred in the remote end while processing the command."}}
```

File: Docs/RunningOnRemoteMachine.md [hunk 1]
```diff
@@ -28,4 +28,26 @@ capabilities.setCapability("app", "Microsoft.WindowsCalculator_8wekyb3d8bbwe!App
...
+- **5.1**.	Setup Batch file to kill any old instances of WinAppDriver - This will be used in the KILLWAD scheduled task later 
+-	*Name* : kill_winappdriver.bat
+-	*Contents* : `taskkill /im WinAppDriver.exe /f`
+- **5.2**.	Setup Batch file to start WinAppdriver  - This will be used in the StartWAD scheduled task later 
+-	*Name* : LaunchWAD.bat
+-	*Contents* : `cmd start /K "C:/Program Files (x86)/Windows Application Driver/WinAppDriver.exe" 10.x.xx.xx 4723/wd/hub`
+-	*Note*: The IP address above (10.x.xx.xx) should be replaced with the local IP address of the server/remote machine
+- **5.3**.	Setup Batch file to logout (without disconnecting) from the remote machine : 
+-	*Name* : logout-rdp.bat
+-	*Contents*: `for /f "skip=1 tokens=3" %%s in ('query user %USERNAME%') do (%windir%\System32\tscon.exe %%s /dest:console C:\Install\QRes.exe /x 1920 /y 1080)`
```
Comment (licanhua) [inline Docs/RunningOnRemoteMachine.md:43]: what QRes.exe?

Comment (robinguptaprovar) [reply Docs/RunningOnRemoteMachine.md:43]: @licanhua - We can setup Qres (http://qres.sourceforge.net/) or another remote vnc tool to change the resolution of the target machine to match the local machine. My solution is loosely derived from here : https://stackoverflow.com/questions/35862217/screen-resolution-changes-for-the-remote-machine-after-safely-closing-the-rdp-se . 

Alternatively the user can skip setting up the destination resolution with QRes .

Comment (smilinrobin) [reply Docs/RunningOnRemoteMachine.md:43]: @licanhua - We can setup Qres (http://qres.sourceforge.net/) or another remote vnc tool to change the resolution of the target machine to match the local machine. My solution is loosely derived from here : https://stackoverflow.com/questions/35862217/screen-resolution-changes-for-the-remote-machine-after-safely-closing-the-rdp-se .

Alternatively the user can skip setting up the destination resolution with QRes .

Comment (licanhua) [inline Docs/RunningOnRemoteMachine.md:43]:
```suggestion
-	*Contents*: `for /f "skip=1 tokens=3" %%s in ('query user %USERNAME%') do (%windir%\System32\tscon.exe %%s /dest:console C:\Install\QRes.exe /x 1920 /y 1080)`
-	*Note*:  Qres (http://qres.sourceforge.net/) is a remote vnc tool to change the resolution of the target machine to match the local machine
```

**Comment by smilinrobin** [comment]
Updated commit with the notes around Qres

**Review by licanhua — APPROVED** [review]

**Comment by licanhua** [comment]
@kat-y and @DHowett Can you help to review and merge it? I think these handy scripts are very useful

File: Docs/RunningOnRemoteMachine.md [hunk 2]
```diff
@@ -28,4 +28,27 @@ capabilities.setCapability("app", "Microsoft.WindowsCalculator_8wekyb3d8bbwe!App
 CalculatorSession = (WindowsDriver)(new WindowsDriver(new URL("http://10.X.X.52:4723/wd/hub"), capabilities));
 CalculatorSession.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
 CalculatorResult = CalculatorSession.findElementByAccessibilityId("CalculatorResults");
- ```
\ No newline at end of file
+ ```
+ 5.WinAppDriver requires the Winappdriver server to be listening on the local host or IP address in order to perform the requisite UI interactions. While Appium generally takes care of this requirement on the local , for running on a server/remote machine, we need to perform a few extra steps as Winappdriver requires a GUI Output and the Winappdriver server running in an interactive shell.
```
Comment (DHowett) [inline Docs/RunningOnRemoteMachine.md]:
```suggestion
 5. WinAppDriver requires the WinAppDriver server to be listening in order to perform the requisite UI interactions. While Appium generally takes care of this requirement on the local machine, running on a server/remote machine requires a few extra steps as WinAppDriver requires an interactive desktop session. 
```

**Review by DHowett — APPROVED** [review]
Thank you for this! I reworded it quite a bit.

**Comment by smilinrobin** [comment]
Thanks @DHowett . 👍

**Comment by lovelyplanet2019** [comment]
I have setup a launch.bat file to call WinAppDriver.exe at test machine IP and port.
`cmd start /K "C:/Program Files (x86)/Windows Application Driver/WinAppDriver.exe" 192.168.2.129 4723` 

Then Jenkins call the :
`C:\Users\8ASO_BUILD\Desktop\Jenkins\workspace\test_RF>CALL C:\prj-ewcom\launch.bat `

At Jenkins console result: the WinAppDriver is exit automatically and cause the robot script failed.

```
C:\Users\8ASO_BUILD\Desktop\Jenkins\workspace\test_RF>cmd start /K "C:/Program Files (x86)/Windows Application Driver/WinAppDriver.exe" 192.168.2.129 4723  
Windows Application Driver listening for requests at: http://192.168.2.129:4723/

Press ENTER to exit.

Exiting...
```

Anyone know how to configure Jenkins properly to run WinAppDriver ?

**Pull request merged**